pkg/process/io.go

```diff
@@ -290,12 +283,7 @@ func NewBinaryIO(ctx context.Context, id string, uri *url.URL) (_ runc.IO, err e
 	}
 	closers = append(closers, r.Close, w.Close)
 
-	cmd := exec.Command(uri.Path, args...)
-	cmd.Env = append(cmd.Env,
-		"CONTAINER_ID="+id,
-		"CONTAINER_NAMESPACE="+ns,
-	)
-
+	cmd := runtime.NewBinaryCmd(uri, id, ns)
```

Comment:
Well. For the dependency, the runtime package depends on `pkg/process` package.  :) The duplicate code is not bad in some case.

Reply:
> the runtime package depends on pkg/process package

Doesn't seem so

Reply:
The `runtime` package is almost used to define interface and the `runtime/{v1,v2}/{runc,shim}` packages share that `pkg/process`. If possible, I would like to see the `NewBinaryCmd` in `pkg` instead of `runtime`